test/interpreter/function-multinomial.spec.ts

```diff
@@ -41,7 +41,7 @@ describe('Function MULTINOMIAL', () => {
   it('coerces to number', () => {
     const engine = HyperFormula.buildFromArray([
       ['=MULTINOMIAL("2",4)'],
-      ['=MULTINOMIAL(B2:C2)', '\'2', 4],
+      ['=MULTINOMIAL(B2:C2)', '2', 4],
```

Comment:
Why a change here? MULTINOMIAL in Excel will return `15` correctly when B2 is `'2`